docs/topics/native/native-improving-compilation-time.md

```diff
@@ -72,6 +71,13 @@ Here are some recommendations for configuring Gradle for better compilation perf
   the build completes successfully. It is possible that these properties were added previously to work around issues that
   have already been fixed.
 
+* **Try incremental compilation of klib artifacts**. With incremental compilation, if your change of the project module
+  causes only a partial recompilation of source code into `klib` artifacts, just a part of the `klib` is further
```

Comment:
But it is the other way around: sources are fully recompiled into klibs, but then the compiler attempts to recompile only some parts of klibs into a binary.

Reply:
Oh, it was copied from What's new, I suggested a change there as well